diesel/src/mysql/types/numeric.rs

```diff
@@ -2,28 +2,23 @@
 pub mod bigdecimal {
     extern crate bigdecimal;
 
-    use std::error::Error;
+    use self::bigdecimal::BigDecimal;
     use std::io::prelude::*;
 
     use mysql::{Mysql, MysqlType};
-
-    use self::bigdecimal::BigDecimal;
-
     use types::{self, FromSql, HasSqlType, IsNull, ToSql, ToSqlOutput};
+    use {deserialize, serialize};
 
     impl ToSql<types::Numeric, Mysql> for BigDecimal {
-        fn to_sql<W: Write>(
-            &self,
-            out: &mut ToSqlOutput<W, Mysql>,
-        ) -> Result<IsNull, Box<Error + Send + Sync>> {
+        fn to_sql<W: Write>(&self, out: &mut ToSqlOutput<W, Mysql>) -> serialize::Result {
             write!(out, "{}", *self)
                 .map(|_| IsNull::No)
                 .map_err(|e| e.into())
```

Comment:
This should be written as `Into::into` 
I think there is a clippy lint for this, but it seems to have this overseen.

Reply:
I don't think existing code that does this is worth touching.